packages/python-packages/doc-warden/warden/enforce_target_file_presence.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from __future__ import print_function
+
+import pathlib
+import os
+import glob
+import fnmatch
+import zipfile
+from .warden_common import get_java_package_roots, get_net_packages, get_python_package_roots, get_js_package_roots, find_alongside_file
```

Comment:
We still use `get_net_packages` instead of `get_net_package_roots`. Does this completely eliminate the usage of `get_net_package_roots`? If so, please delete it from `warden_common`.